pyproject.toml

```diff
@@ -28,11 +29,11 @@ license = "Apache-2.0"
 [tool.poetry.dependencies]
 python = "^3.8"
 dagster = "^1.4.0"
-polars = ">=0.17.0"
+polars = ">=0.20.1"
```

Comment:
Let's not change the lower polars constraint. We don't want to force an update for users as it can break their code.

Reply:
To be clear, we do want to update the dev polars version pinned in poetry.lock. This can be done via "poetry update" command.